sandbox/registration-processor-mz.properties

```diff
@@ -384,24 +384,49 @@ registration.processor.reprocess.months=*
 #schedular weeks configuration
 registration.processor.reprocess.days_of_week=*
 
+#--------------workflow-manager-service----------------
+
 # WorkflowAction api id
-mosip.regproc.workflow.action.api-id=mosip.registration.processor.workflow.action
+mosip.regproc.workflow-manager.action.api-id=mosip.registration.processor.workflow.action
 
 # WorkflowAction api version
-mosip.regproc.workflow.action.version=1.0
+mosip.regproc.workflow-manager.action.version=1.0
 
 # WorkflowSearch api id
-mosip.regproc.workflow.search.api-id=mosip.registration.processor.workflow.search
+mosip.regproc.workflow-manager.search.api-id=mosip.registration.processor.workflow.search
 
 # WorkflowSearch api version
-mosip.regproc.workflow.search.version=v1
+mosip.regproc.workflow-manager.search.version=v1
 
 #WorkflowAction service  beginning stage name
-mosip.regproc.workflow.action.resumefrombeginning.stage=PacketUploaderStage
+mosip.regproc.workflow-manager.action.resumefrombeginning.stage=PacketUploaderStage
 
 #WorkflowAction service  websub topic name
 mosip.regproc.workflow.complete.topic=registration_processor_workflow_completed_event
 
+#Number of the reprocess records to be fetched at a time
+registration.processor.workflow-manager.action.job.fetchsize=100
```

Comment:
can you make these properties also to follow mosip.regproc.workflow-manager ?

Reply:
fixed